package exception.handling;

public class Main01 {
    public void finalize(){
        System.out.println("finalize called");
    }
}
